api/go.mod

```diff
@@ -1,9 +1,11 @@
 module github.com/photoview/photoview/api
 
-go 1.22.0
+go 1.22.5
+
+toolchain go1.23.1
```

Comment:
Is it ok that the `toolchain` version is not synced with the `go` version?

Reply:
It's OK. `go` means the feature in the code, `toolchain` means compiler version. As we are using `golang-1.23.1` image, it's fine.

Reply:
It's also managed by `go mod tidy`.